infra/terraform/bootstrap/azure_ad.tf

```diff
@@ -9,7 +9,7 @@ resource "azuread_group" "dbadminsaadgroup" {
   display_name  = "${var.prefix}-${var.environment}-omop-sql-server-admins"
   mail_nickname = "${var.prefix}-${var.environment}-omop-sql-server-admins"
   owners = toset([
-    data.azuread_client_config.current.object_id,
+    sensitive(data.azuread_client_config.current.object_id),
```

Comment:
what does `sensitive()` do?

Reply:
https://www.terraform.io/language/functions/sensitive

`sensitive` takes any value and returns a copy of it marked so that Terraform will treat it as sensitive, with the same meaning and behavior as for [sensitive input variables](https://www.terraform.io/language/values/variables#suppressing-values-in-cli-output).